README.md

```diff
@@ -46,6 +46,7 @@ On Windows you will need [Microsoft C++ Build Tools](https://visualstudio.micros
 2. Install the package from [PyPI](https://pypi.org):
 
 ```bash
+conda create -n my_environment_name python=3.6
```

Comment:
Is there any reason for this line?
1. we don't depend on conda, the user may prefer venv
2. the version need not be 3.6, it could be 3.7 (and we say this a couple of lines above).